task/build-helm-chart-oci-ta/0.2/README.md

```diff
@@ -0,0 +1,257 @@
+# Build Helm Chart OCI TA Task
+
+This Tekton task packages and pushes a Helm chart to an OCI repository with support for
+image substitution.
+
+## Features
+
+- **Chart Packaging**: Packages Helm charts with semver-compatible versioning
+- **OCI Push**: Pushes packaged charts to OCI registries
+- **Image Substitution**: Replaces source images with target images in chart templates
+- **Git-based Versioning**: Uses git tags to determine chart versions
+- **Trusted Artifacts**: Uses trusted artifacts for source code access
+
+## Parameters
+
+| Parameter | Description | Default | Required |
+|-----------|-------------|---------|----------|
+| `IMAGE` | Full image reference with tag (e.g., quay.io/redhat-user-workloads/konflux-vanguard-tenant/caching/squid:on-pr-{{revision}}) | - | Yes |
+| `COMMIT_SHA` | Git commit SHA to build chart for | - | Yes |
+| `SOURCE_ARTIFACT` | The Trusted Artifact URI pointing to the artifact with the application source code | - | Yes |
+| `SOURCE_CODE_DIR` | Path relative to workingDir where code was pulled | `source` | No |
+| `CHART_CONTEXT` | Path relative to SOURCE_CODE_DIR where chart is located | `dist/chart/` | No |
+| `VERSION_SUFFIX` | Suffix to be added to the version string | `""` | No |
+| `TAG_PREFIX` | Prefix for version tag matching | `helm-` | No |
+| `IMAGE_MAPPINGS` | JSON array of image mappings for substitution. Target images will have tags automatically set to VERSION_SUFFIX-COMMIT_SHA format. Substitutions occur in templates/ and values files. | `[]` | No |
```

Comment:
There is no automated tag manipulation other then the regular assumption that no-tag == "latest"